source/type/type.h

```diff
@@ -708,6 +710,10 @@ struct StopTime {
     uint32_t arrival_time = 0; ///< seconds since midnight
     uint32_t departure_time = 0; ///< seconds since midnight
 
+    /// Representing times used in Raptor, taking into account boarding and alighting process duration
+    uint32_t boarding_time = 0; ///< seconds since midnight
```

Comment:
Adding two int32 to StopTime is a risk for RAM use, we have to measure its impact (and probably Kisio should do that on it's biggest instances)

- [ ] Monitor RAM use increase (on fr-idf?)